var/spack/repos/builtin/packages/thrift/package.py

```diff
@@ -45,6 +45,15 @@ class Thrift(Package):
 
     # Variant dependencies
     extends('python', when='+python')
+    depends_on('py-setuptools', type=('build', 'run'), when='+python')
+    depends_on('py-six@1.7.2:', type=('build', 'run'), when='@0.10.0:+python')
+    depends_on('py-tornado', type=('build', 'run'), when='+python')
+    depends_on('py-twisted', type=('build', 'run'), when='+python')
+    depends_on('py-zope-interface', type=('build', 'run'), when='+python')
+    depends_on('py-ipaddress', type=('build', 'run'), when='^python@2')
```

Comment:
I would actually use:
```suggestion
    depends_on('py-ipaddress', type=('build', 'run'), when='+python ^python@2')
```
Same below. I'm not sure if `^python` will pick up direct python deps, or transient python deps where `^python` is somewhere else in the DAG.